src/grammar.coffee

```diff
@@ -154,8 +154,19 @@ grammar =
     o 'CSX_TAG',                                -> new CSXTag $1
   ]
 
+  # PropertyNumber is `NumberLiteral` `Accessor`, e.g. obj.42
```

Comment:
@zdenko this explanation is a bit confusing/misleading, when I read
```
`NumberLiteral` `Accessor`
```
I think it means that `PropertyNumber` is the grammar sequence `NumberLiteral Accessor`

Maybe say eg
```
PropertyNumber is a `NumberLiteral` as an access, e.g. obj.42
```